OrbitGl/GlCanvas.cpp

```diff
@@ -343,6 +349,21 @@ float GlCanvas::ScreenToWorldWidth(int width) const {
   return (static_cast<float>(width) / static_cast<float>(GetWidth())) * world_width_;
 }
 
+Vec2 GlCanvas::WorldToScreen(Vec2 world_pos) const {
+  Vec2 screen_pos;
+  screen_pos[0] = (world_pos[0] - world_top_left_x_) / world_width_ * GetWidth();
+  screen_pos[1] = (world_top_left_y_ - world_pos[1]) / world_height_ * GetHeight();
+  return screen_pos;
+}
```

Comment:
I think we should use floorf(...), to be consistent with how TextRenderer positions glyphs. Also this will make it easy to spot screen space coordinates as their decimal part will always be 0.f. Wdyt? I was tempted to suggest returning integers for screen space, as GetWidth() and GetHeight() do, but it's true that we probably always convert to float. If there is a need for an "int" version, we should probably add it as a separate function.

Reply:
Good point, I've added that. Regarding returning `int`, and also related to your comment below: I agree, and initially, I wanted to make this return an int-vector. But I've decided against it for consistency since we're using Vec2 for screen-space in a lot of other places as well.

But I agree that it would be nice to change this all to int at some point.